resolve/resolve.go

```diff
@@ -715,6 +718,14 @@ func (r *resolver) expr(e syntax.Expr) {
 				r.expr(arg)
 			}
 		}
+		if p >= 256 {
```

Comment:
Comment:

// Fail gracefully if compiler-imposed limit is exceeded.

Reply:
Done.